Robust.Shared/Physics/Components/PhysicsComponentState.cs

```diff
@@ -25,11 +24,35 @@ namespace Robust.Shared.Physics.Components;
     public readonly bool FixedRotation = FixedRotation;
     public readonly BodyStatus Status = Status;
 
-    public readonly Vector2 LinearVelocity = LinearVelocity;
-    public readonly float AngularVelocity = AngularVelocity;
+    public Vector2 LinearVelocity = LinearVelocity;
+    public float AngularVelocity = AngularVelocity;
     public readonly BodyType BodyType = BodyType;
 
     public readonly float Friction = Friction;
     public readonly float LinearDamping = LinearDamping;
     public readonly float AngularDamping = AngularDamping;
 }
+
+[Serializable, NetSerializable]
+public sealed class PhysicsDeltaState : IComponentState, IComponentDeltaState
+{
+    public Vector2 LinearVelocity;
+    public float AngularVelocity;
+
+    public bool FullState => false;
+
```

Comment:
Okay you know what I have NFI how deltas are supposed to work, I guess I mark this as FullState = true and shit's gravity?

```suggestion

    public bool FullState => true;

```